### ISSUE TITLE
UV Map corruption on GLB import

### ISSUE THREAD

**Issue by Sandstorm-mt** [godotengine/godot#100172]
### Tested versions

Godot 4.3

### System information

Forward+, Windows 10, I3 4190

### Issue description

When importing GLB models into Godot, textures seem stretched or corrupted.

Model in Blender(imported GLB)
![Image](https://github.com/user-attachments/assets/62092cac-6c9a-4aff-97e9-32fb554169c1)

Model in Godot Editor
![Image](https://github.com/user-attachments/assets/22ab68bb-f045-4f94-85f3-47220bdd0865)

btw materials aren`t corrupted:
![Image](https://github.com/user-attachments/assets/7359a797-58d5-42aa-a60b-6d384f5e43af)
![Image](https://github.com/user-attachments/assets/6df8bb12-d52f-4685-9740-e63e3c232da5)
![Image](https://github.com/user-attachments/assets/5acfb1f2-d928-4201-9431-dbd504aa2650)

### Steps to reproduce

Import model into the editor and see corruption of UV

[axe.zip](https://github.com/user-attachments/files/18052186/axe.zip)

### Minimal reproduction project (MRP)

N/A

**Comment by fire** [comment]
Try higher quality texture compression

**Comment by Uumutunal** [comment]
On your model you have 2 UV sets, UVMap and UVMap.001, and in materials you're using UVMap.001, when you import it to godot it automatically uses the first UV set (UVMap). This is why you don't see the textures, if you remove the unused UV set from the model it works.

Right one is imported without removing the unused UV set, left one i after I removed it.
![Image](https://github.com/user-attachments/assets/d34a99b5-a310-40d8-8a84-6fd6ecf7bd1b)

**Comment by TheSofox** [comment]
Good catch. I had the exact same problem with another model (on v4.4.1.stable.official [49a5bc7b6] ) and when looking it up I found this bug report and confirmed I had two UVmaps. Deleting one fixed the problem.

Even though the user has a clear workaround, this does seem like an bug. It makes intuitive sense that Godot would use whichever UV map was used to display the model, not just choose the first one in the list. Even Windows 3D Viewer uses the right UV map by default.

<img width="1003" height="841" alt="Image" src="https://github.com/user-attachments/assets/6595fb04-7951-49f3-8771-f16e13fac54d" />

**Comment by fire** [comment]
Nothing for this cycle but for 4.6 we could start looking into this.

**Comment by aaronfranke** [comment]
Nothing to look into, it's already solved. PR https://github.com/godotengine/godot/pull/96748 fixes this.
